presto-common/src/main/java/com/facebook/presto/common/resourceGroups/QueryType.java

```diff
@@ -26,6 +26,7 @@
     ANALYZE(5),
     INSERT(6),
     SELECT(7),
+    SESSION_TRANSACTION_DEPENDENT_DATA_DEFINITION(8),
```

Comment:
why so long; just `TRANSACTION`?

Reply:
Agree name is too long. We have two types of statements, for transaction_control and session_transaction.

Reply:
Actually, while thinking about again, I like the name TRANSACTION. It's simple and clear.